includes/class-wc-stripe-payment-method-configurations.php

```diff
@@ -159,6 +159,8 @@ private static function get_payment_method_configuration_from_stripe() {
 			}
 		}
 
+		// If we don't have a Payment Method Configuration that inherits from the WooCommerce Platform, disable Payment Method Configuration sync.
+		self::disable_payment_method_configuration_sync();
```

Comment:
I think we need to guard against cases where we receive `[]` or `null` from the `get_payment_method_configurations()` call, which could easily be the case if we've been rate limited.

Reply:
Also, might it make sense to add a lower-level cache to the raw `get_payment_method_configurations()` API calls? (I don't think this blocks the logic here, but that feels like it could also avoid unnecessary API calls.)

Reply:
> I think we need to guard against cases where we receive `[]` or `null` from the `get_payment_method_configurations()` call, which could easily be the case if we've been rate-limited.

I think we should default to disabling sync if anything unexpected happens, and once we stabilize the rate limiting problem, we can attempt to re-enable sync (with a retry limit or similar, like try 4 times with an interval of 8n hours)

Reply:
Noting that @diegocurbelo and I discussed this yesterday, and we landed on adding a comment as a starting point.